addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -603,7 +603,6 @@ async function start(param0 = {}) {
     registerCleanup(async () => {
         await webClient.env.services.messaging.modelManager.messagingInitializedPromise;
         webClient.env.services.messaging.modelManager.destroy();
-        webClient.env.services.legacy_bus_service.destroy();
```

Comment:
This was here to avoid memory leaks in tests, are you sure it can just be removed?

Reply:
Yes, listeners are registered through `browser` now, which automatically remove listeners after each test.